builtin/settingtypes.txt

```diff
@@ -634,6 +634,9 @@ crosshair_color (Crosshair color) string (255,255,255)
 #    Crosshair alpha (opaqueness, between 0 and 255).
 crosshair_alpha (Crosshair alpha) int 255 0 255
 
+#    Maximum number of recent chat items to show
```

Comment:
Should also say 'lines'.

Reply:
If changing, then to "messages", as they may exceed one line depneding on font size, screensize and character limit.

Reply:
Ah, maybe 'messages' is better than 'lines' then?